cumulus/primitives/core/src/lib.rs

```diff
@@ -395,4 +399,10 @@ sp_api::decl_runtime_apis! {
 		/// we are collecting the collation info for.
 		fn collect_collation_info(header: &Block::Header) -> CollationInfo;
 	}
+
+	/// Runtime api to fetch the claim queue offset.
+	pub trait FetchClaimQueueOffset {
```

Comment:
IMO we should probably tag the `UnincludedSegment` api as deprecated and then merge both of the functions into a new `ParachainConsesusExt` or similar. Or is this a dumb idea?

Reply:
I don't think this would be very useful. I think it's nice to have this separation.
Also, there is this https://github.com/paritytech/polkadot-sdk/blob/655382fa87cd82f12920e316af63084ae098a4e3/cumulus/polkadot-parachain/polkadot-parachain-lib/src/common/mod.rs#L62 if we need a trait that groups all runtime APIs